pkg/cloudinit/scripts/install.sh

```diff
@@ -2,7 +2,19 @@
 
 ## Assumptions:
 ## - /capi/etc/snap-track contains the snap track that matches the installed Kubernetes version, e.g. "v1.30.1" -> "1.30-classic/stable"
```

Comment:
I think if the `track` needs to be replaced with `channel` according to the comments above, this one should be changes as well.